docs/en/api/POSIX-API.md

```diff
@@ -73,181 +73,363 @@ For example, running the following commands from the `${ALLUXIO_HOME}` directory
 
-Enable FUSE on worker by setting `alluxio.worker.fuse.enabled` to `true` in the `${ALLUXIO_HOME}/conf/alluxio-site.properties`:
+Enable via Fuse mount command:
+```console
+$ ${ALLUXIO_HOME}/integration/fuse/bin/alluxio-fuse mount -o attr_timeout=600,entry_timeout=600 /mnt/people /people
+```
 
+Enable via Alluxio configuration before mounting, edit the `${ALLUXIO_HOME}/conf/alluxio-site.properties`:
 ```config
-alluxio.worker.fuse.enabled=true
+alluxio.fuse.mount.options=entry_timeout=600,attr_timeout=600
 ```
 
-By default, Fuse on worker will mount the Alluxio root path `/` to default local mount point `/mnt/alluxio-fuse` with no extra mount options.
-You can change the alluxio path, mount point, and mount options through Alluxio configuration:
+If enough memory resources are available, recommend setting the timeout to a large value to cover the whole training period to avoid refreshing cache.
+If not, even a 1-minute to 10-minute kernel metadata cache may significantly improve the overall metadata read performance and/or data read performance.
+
+Kernel metadata cache for a single file takes around 300 bytes (up to 1KB), 3GB (up to 10GB) for 10 million files.
 
+Recommend to use kernel metadata cache when launching Fuse process on plain machine (not via containerized environment)
+or when FUSE container has enough memory resources and thus will not be killed unexpectedly
+when memory usage (Fuse process memory + Fuse kernel cache) exceeds the configured container memory limit.
+
+  {% endnavtab %}
+  {% navtab Userspace Metadata Cache Configuration %}
+
+Userspace metadata cache can be enabled via setting Alluxio configuration in `${ALLUXIO_HOME}/conf/alluxio-site.properties`
+before mounting:
 ```config
-alluxio.fuse.mount.alluxio.path=<alluxio_path>
-alluxio.fuse.mount.point=<mount_point>
-alluxio.fuse.mount.options=<list of mount options separated by comma>
+alluxio.user.metadata.cache.enabled=true
+alluxio.user.metadata.cache.expiration.time=2h
+alluxio.user.metadata.cache.max.size=2000000
+```
+
+The metadata is cached in the AlluxioFuse Java process heap so make sure `cache.max.size * 2KB * 2 < AlluxioFuse process maximum memory allocated`.
+For example, if AlluxioFuse is launched with `-Xmx=16GB` and metadata cache can use up to 8GB memory, then `cache.max.size` should be smaller than 4 million.
+The default max cache size is `100000` which takes about `200MB` heap space.
+
+When the data is updated via other Alluxio clients, the metadata cache needs to be updated otherwise it will be stale until reaches `alluxio.user.metadata.cache.expiration.time` (default is `10min`).
+Metadata cache can be invalidated manually through Alluxio FUSE shell which is enabled by setting `alluxio.fuse.special.command.enabled=true` in `${ALLUXIO_HOME}/conf/alluxio-site.properties`
+before mounting FUSE.
+
+Run the Fuse shell command to clear all the cached metadata：
+```console
+$ ls -l /mnt/alluxio-fuse/.alluxiocli.metadatacache.dropAll
```

Comment:
Recommend replacing `/mnt/alluxio-fuse` with <mount_point> since this is an example